app/views/handlers/question.py

```diff
@@ -210,22 +225,20 @@ def get_return_to_hub_url(self):
         ):
             return url_for(".get_questionnaire")
 
-    def evaluate_and_update_section_status_on_list_change(self, list_name):
+    def add_dependent_sections_for_list(self, list_name):
```

Comment:
Minor, for consistency with other method names, maybe call it `capture` instead of `add?.